### DISCUSSION TITLE
Contestar en castellano

### DISCUSSION THREAD

**Discussion by jogugil** [imartinez/privateGPT#693] [General]
Por qué si le hago una pregunta en castellano me contesta en ingles. He ingestado tres documentos en castellano, porqué me habla en ingles?. Cómo configurarlo para que me hable en español?

**Comment by noguerad** [comment]
Buenas @jogugil , si la primera órden que le das es "From now on reply only in spanish" te responde sólo en español.
Yo tampoco he conseguido que lea y aprenda documentos pdf en español, por ahora únicamente en inglés, pero espero haberte ayudado con lo primero.
